web/frontend/src/components/slam-2d-render.vue

```diff
@@ -204,13 +146,11 @@ const disposeScene = () => {
   scene.clear();
 };
 
-const updatePose = async (newPose: commonApi.Pose) => {
+const updatePose = (newPose: commonApi.Pose) => {
```

Comment:
why does this no longer need to be async

Reply:
We were loading / parsing svgs in this function if a marker had not yet been created using `THREE`'s `SVGLoader`. After moving us to use textures instead that's no longer necessary. The planes that the icon texture are rendered on are now created at startup, and the `TextureLoader` will background-load the image maps.